docs/edc/EDC_BPDM_Usage.postman_collection.json

```diff
@@ -487,32 +6335,52 @@
     }
   ],
   "variable": [
+    {
+      "key": "SHARING_MEMBER_EDC_BASE_URL",
```

Comment:
This is really great, depending on environment we can adapt the sharing member EDC and gate EDC base urls